ui/admin/app/routes/scopes/scope/roles/role/manage-scopes/manage-org-projects.js

```diff
@@ -39,12 +39,19 @@ export default class ScopesScopeRolesRoleManageScopesManageOrgProjectsRoute exte
       scope_id: [{ equals: org_id }],
     };
 
-    const projectScopes = await this.store.query('scope', {
-      scope_id: org_id,
-      query: { search, filters },
-      page,
-      pageSize,
-    });
+    // All scopes have already been pre-loaded in the `scopes.scopes.roles.role` route.
+    // Therefore, we can simply peek into indexedDB to retrieve results.
+    const options = { peekIndexedDB: true };
+    const projectScopes = await this.store.query(
```

Comment:
This wasn't working for me if a user has never loaded anything into indexed db and goes to the manage scopes for projects directly. I think it's because the scopes are being loaded in `role.index` and not `role` which means it's not considered a child of the route so it won't pre-load the scopes and doesn't return any results.

You could either load the scopes in `role` or honestly we could just include the `recursive` flag which would share the same query anyways since the token that's being stored would just be re-used

Reply:
well thats good to know

good catch! im leaning towards reusing the same query instead. there could be more scenarios i missed and that seems to be the safer option.